pkgs/applications/window-managers/eww/default.nix

```diff
@@ -22,9 +24,9 @@ rustPlatform.buildRustPackage rec {
 
   cargoHash = "sha256-dC7yVJdR7mO0n+sxWwigM1Q4tbDv5ZuOINHHlUIPdA0=";
 
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [ pkg-config wrapGAppsHook ];
 
-  buildInputs = [ gtk3 gdk-pixbuf ] ++ lib.optional withWayland gtk-layer-shell;
+  buildInputs = [ gtk3 gdk-pixbuf librsvg ] ++ lib.optional withWayland gtk-layer-shell;
```

Comment:
does that mean we can remove gdk-pixbuf here?

Reply:
Probably not, as that handles PNG/JPG/non-vector images.
I'll test and report.

Reply:
There doesn't seem to be any difference, should I remove it?

Reply:
yes please, thanks for testing!